src/main/java/com/rakovets/course/java/core/practice/oop/classesandobjects/task_4/model/Atm.java

```diff
@@ -0,0 +1,83 @@
+package com.rakovets.course.java.core.practice.oop.classesandobjects.task_4.model;
+
+public class Atm {
+    private int numberBanknotes100;
+    private int numberBanknotes50;
+    private int numberBanknotes20;
+    private int amount;
+
+    public Atm(int numberBanknotes100, int numberBanknotes50, int numberBanknotes20) {
+        this.numberBanknotes100 = numberBanknotes100;
+        this.numberBanknotes50 = numberBanknotes50;
+        this.numberBanknotes20 = numberBanknotes20;
+    }
+
+    public void addBanknotes100(int numberBanknotes100) {
+        this.numberBanknotes100 += numberBanknotes100;
+    }
+
+    public void addBanknotes50(int numberBanknotes50) {
+        this.numberBanknotes50 += numberBanknotes50;
+    }
+
+    public void addBanknotes20(int numberBanknotes20) {
+        this.numberBanknotes20 += numberBanknotes20;
+    }
+
+
+    public void isPossible(int amount) {
```

Comment:
Why doesn't this method return anything?

Reply:
Because in this method i implemented output with `System.out.print`. Or is it better to use return in this case?